packages/sources/css/src/components/badge/src/index.css

```diff
@@ -0,0 +1,85 @@
+/* Design tokens */
+@import '@vtmn/css-design-tokens/src/colors';
+@import '@vtmn/css-design-tokens/src/spacings';
+@import '@vtmn/css-design-tokens/src/typography';
+
+.vtmn-badge {
+  font-family: var(--vtmn-typo_font-family);
+  font-size: var(--vtmn-typo_text-2-font-size);
+  display: inline;
+}
+
+.vtmn-badge:focus-visible {
+  outline: none;
+  box-shadow: 0 0 0 rem(4px) var(--vtmn-color_white),
+    0 0 0 rem(6px) var(--vtmn-color_black);
+}
+
+.vtmn-badge span[class^='vtmx-'] {
```

Comment:
Badge should also work with `img` and `svg`for CSS styles

Reply:
From now, badge works with svg iconq